oss_src/unity/python/sframe/cython/cy_cpp_utils.pyx

```diff
@@ -52,3 +52,28 @@ cdef string _attempt_cast_str_to_cpp(py_s) except *:
 
     # Okay, none of these worked, so error out.
     raise TypeError("Type '%s' cannot be interpreted as str." % str(type(py_s)))
+
+
+cdef bint _cpp_to_str_py3_decode_enabled = True
+
+cpdef disable_cpp_str_decode():
+    global _cpp_to_str_py3_decode_enabled
+    assert _cpp_to_str_py3_decode_enabled == True
+    _cpp_to_str_py3_decode_enabled = False
+
+cpdef enable_cpp_str_decode():
+    global _cpp_to_str_py3_decode_enabled
+    assert _cpp_to_str_py3_decode_enabled == False
+    _cpp_to_str_py3_decode_enabled = True
+
```

Comment:
note that this is kind of iffy. Especially since the flag could be set, then a ctrl-C / some other failure happens which cause the flag to stay set. A "with xxx" mechanism is preferred.